### PR TITLE
feat: Add webpack course

### PR DESCRIPTION
**Pull request by Hari70a**
## What does this PR do?
- Add webpack course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Hari70a** [comment]
@ImVector,
 
 Based on your suggestion in https://github.com/EbookFoundation/free-programming-books/pull/7971,  I am raising this new MR, which closes https://github.com/EbookFoundation/free-programming-books/issues/7972. Kindly review it.

cc: @davorpa & @eshellman

**Review by LuigiImVector — COMMENTED** [review]
we try to avoid creating a new section for only one resource, try inserting it into an existing one

**Review by LuigiImVector — COMMENTED** [review]
add webpack to the index

**Comment by Hari70a** [comment]
> add webpack to the index

Done @LuigiImVector . Thank you

**Comment by eshellman** [comment]
Shouldn't Webpack should go under Javascript?

**Comment by Hari70a** [comment]
> Shouldn't Webpack should go under Javascript?

@eshellman ,

To my understanding, Webpack is created with JS, but really a JS framework. I believe adding to Index is fine,

Kindly correct me if I am wrong.

cc: @LuigiImVector & @davorpa

**Comment by davorpa** [comment]
> > Shouldn't Webpack should go under Javascript?
> 
> @eshellman ,
> 
> To my understanding, Webpack is created with JS, but not really a JS(UI) framework. I believe adding to Index is fine,
> 
> Kindly correct me if I am wrong.
> 
> cc: @LuigiImVector & @davorpa

Webpack, Parcel... is a bundle processor written/pluggable in javascript. So maybe better have a category `JavaScript -> Module Bundlers`

**Comment by Hari70a** [comment]
> 



> > > Shouldn't Webpack should go under Javascript?
> > 
> > 
> > @eshellman ,
> > To my understanding, Webpack is created with JS, but not really a JS(UI) framework. I believe adding to Index is fine,
> > Kindly correct me if I am wrong.
> > cc: @LuigiImVector & @davorpa
> 
> Webpack, Parcel... is a bundle processor written/pluggable in javascript. So maybe better have a category `JavaScript -> Module Bundlers` to glob all

Gotcha @davorpa ,

Adding directly to `Javascript` feels odd to me. Now I am clear with your suggestion.

Thank you

**Comment by Hari70a** [comment]
Updated based on your suggestions,

Please let me know if anything pending from me. I could see this `Awaiting for changes` label and no further actions

**Comment by Hari70a** [comment]
Please let me know if you are expecting any changes from me.

Thank You.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -772,6 +772,8 @@
 * [Learn modern JavaScript](https://scrimba.com/learn/es6) - Beau Carnes (Scrimba) (Scrimba account *required*)
 * [Learn to Program in Javascript: Beginner to Pro](https://www.udemy.com/course/programming-in-javascript) - Raghavendra Dixit (Udemy)
 * [learn:query](https://learnquery.infinum.com)
+* Module Bundlers
```
Comment (eshellman) [inline courses/free-courses-en.md:775]: not sure what you meant to do here

Comment (Hari70a) [reply courses/free-courses-en.md:775]: Based on the previous discussions, I added this in Module Bundlers of Javascript section. Please let me know if I need to update anything

**Comment by eshellman** [comment]
added one resource from this PR in #9177. The other is neither a book or a course

**Pull request closed without merging**